app/src/main/java/com/github/kulebin/myfoursquareapp/VenueDetailFragment.java

```diff
@@ -0,0 +1,74 @@
+package com.github.kulebin.myfoursquareapp;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.ActionBar;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.github.kulebin.myfoursquareapp.imageLoader.IImageLoader;
+import com.github.kulebin.myfoursquareapp.presenter.VenueDetailPresenter;
+import com.github.kulebin.myfoursquareapp.view.CompleteVenueDisplayData;
+import com.github.kulebin.myfoursquareapp.view.VenueDetailContract;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
```

Comment:
This library helps to reduce the amount of boilerplate code. On compiler time it replaces annotated views with common findViewById methods